js/private/js_image_layer.bzl

```
@@ -18,9 +18,19 @@ load("@bazel_skylib//lib:paths.bzl", "paths")
 
 _DOC = """Create container image layers from js_binary targets.
 
-js_image_layer supports transitioning to specific platform for cross-compiling.
+By design, js_image_layer doesn't have any preference over rule which assembles the container image. 
+However, this comes with a few quirks; the downstream rule, `oci_image` or `container_image` in this case,
+has to set proper `workdir` and `cmd` to make the container work.
 
-A partial example using rules_oci with transition to linux/amd64 platform.
+Usually, `cmd` needs to be set to **/** `[ root of <js_image_layer> ]` **/** `[ relative path to BUILD file from WORKSPACE or package_name() ] / [ name of <js_binary> ]`, 
```

```suggestion
Usually `cmd` needs to be set to **/** `[ root of <js_image_layer> ]` **/** `[ relative path to BUILD file from WORKSPACE or package_name() ] / [ name of <js_binary> ]`, 
```

I'm not sure what the bold "/"s mean.. is that a syntax or trying to highlight something?

---

those are not supposed to look this bad. I'll revert those.